### PR TITLE
chore: Enable chatops bot. Fixes #7757

### LINKED ISSUE #7757: Add a bot to help triage issues
We want to plan a roadmap and would like to have issues labelled with components.

It would be good to label issues by version too.

### PR DESCRIPTION
**Pull request by alexec**
Signed-off-by: Alex Collins <alex_collins@intuit.com>
Fixes #7757
Outcomes:

* All issues have a component added. So we can search by component.
* All bugs have a version added, so we can search by version
* More bug/regressions get raised with the correct details.
* More PRs raised in reviewable/mergeable state.
* Issues are stale and closed faster.
* Source code is checkout correctly.

### CODE REVIEW

File: .github/governance.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        comment: |
+          @$AUTHOR: There are no 'kind' label on this.
+
+          * `/kind bug`
+          * `/kind enhancement`
+          * `/kind tech-debt`
+
+    - prefix: component
+      list:
+        - component:cli
```
Comment (alexec) [inline .github/governance.yml:21]: they use `area` themselves,

that is easy to type, "component" is a long work to type, maybe "comp" or "area" better?

Comment (terrytangyuan) [reply .github/governance.yml:21]: area sounds better

File: .github/governance.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          * `/component component:cli`
+          * `/component component:controller`
+          * `/component component:docs`
+          * `/component component:executor`
+          * `/component component:plugins`
+          * `/component component:server`
+          * `/component component:sdks`
+          * `/component component:ui`
+
+    - prefix: v
```
Comment (terrytangyuan) [inline .github/governance.yml:43]: Do bug reports still require versioning info then?

Comment (alexec) [reply .github/governance.yml:43]: yes

File: .github/governance.yml (hunk 1 above)
Comment (alexec) [reply .github/governance.yml:21]: I'm keen to keep consistent with Argo CD, and I don't see Argo CD changing, so not sure what to do here.

File: .github/stale.yml [hunk 3]
```diff
@@ -1,15 +1,15 @@
 # https://probot.github.io/apps/stale/
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 60
+daysUntilStale: 7
```
Comment (alexec) [inline .github/stale.yml:3]: if too rapid, we will change to another value

**Review by sarabala1979 — APPROVED** [review]

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**